### PR TITLE
Update jenkinsfile

### PR DESCRIPTION
**Pull request by cobylu**
Silencing sensitive console outputs in jenkins

### CODE REVIEW

**Comment by cobylu** [comment]
@hankjtaylor @samratjha96 Ready for Code Review

**Comment by hankjtaylor** [comment]
LGTM!!!!

**Comment by samratjha96** [comment]
I'm questioning why even have two methods at this point and not just have shNoTraceNoOutput to be on the safe side like in the one instance where the output of the command actually spilled the password we were suppressing in the command. Are there instances where shNoTrace is used where the output actually contains any valuable information for future debugging? Or is it just not worth the chance and we should standardize shNoTraceNoOutput? @hankjtaylor @cobylu

File: Jenkinsfile [hunk 1]
```diff
@@ -5,11 +5,11 @@ pipeline {
       steps {
         sh "#!/bin/bash"
```
Comment (hankjtaylor) [inline Jenkinsfile]: if we don't think there is any need to echo out every command we do, i think we could even put the silencing statement right here, and then make a function that allows you to print certain commands

**Comment by samratjha96** [comment]
LGTM :trollface:

**Pull request merged**